Catappult/appcoins-wallet-android
# [ESKL-359] Feature/eskl 359 oem id check

**What does this PR do?**

   Adds check to verify if the Wallet app was installed through an eskills campaign. This is done by verifying if the current wallet app has an E-skills oem id. The downloader and installer should be merged before this (https://github.com/Catappult/appcoins-wallet-android/pull/693).

**Database changed?**

   No


**How should this be manually tested?**

  Inject OEM id on walllet apk -> install wallet -> verify if the displayed home screen is the e-skills version (game carrousel, allows game installation without leaving the wallet, etc.).

**What are the relevant tickets?**

  Tickets related to this pull-request: [ESKL-359](https://aptoide.atlassian.net/browse/ESKL-359)

**Code Review Checklist**



[ESKL-359]: https://aptoide.atlassian.net/browse/ESKL-359?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
